From 500156964cede873a4f83c0c9a74d536f12d646e Mon Sep 17 00:00:00 2001
From: Lee Chee Yang <chee.yang.lee@intel.com>
Date: Fri, 19 Jun 2020 16:43:29 +0800
Subject: [PATCH] no life-mngr

dont build life-mngr in acrn-tools.
life-mngr daemon is for User VM only. 
it should be saperate from acrn-tools which for Service VM.

Upstream-Status: Inappropriate

Signed-off-by: Lee Chee Yang <chee.yang.lee@intel.com>
---
 misc/Makefile | 11 +++++------
 1 file changed, 5 insertions(+), 6 deletions(-)

diff --git a/misc/Makefile b/misc/Makefile
index 1a510ad3..a65023ed 100644
--- a/misc/Makefile
+++ b/misc/Makefile
@@ -14,11 +14,11 @@ else
   endif
 endif
 
-.PHONY: all acrn-crashlog acrnlog acrn-manager acrntrace acrnbridge life_mngr
+.PHONY: all acrn-crashlog acrnlog acrn-manager acrntrace acrnbridge
 ifeq ($(RELEASE),n)
-all: acrn-crashlog acrnlog acrn-manager acrntrace acrnbridge life_mngr
+all: acrn-crashlog acrnlog acrn-manager acrntrace acrnbridge
 else
-all: acrn-manager acrnbridge life_mngr
+all: acrn-manager acrnbridge
 endif
 
 acrn-crashlog:
@@ -50,10 +50,9 @@ clean:
 
 .PHONY: install
 ifeq ($(RELEASE),n)
-install: acrn-crashlog-install acrnlog-install acrn-manager-install acrntrace-install acrnbridge-install \
-	acrn-life-mngr-install
+install: acrn-crashlog-install acrnlog-install acrn-manager-install acrntrace-install acrnbridge-install
 else
-install: acrn-manager-install acrnbridge-install acrn-life-mngr-install
+install: acrn-manager-install acrnbridge-install
 endif
 
 acrn-crashlog-install:
-- 
2.25.1
